zcash_primitives/Cargo.toml

```diff
@@ -16,7 +16,7 @@ edition = "2018"
 all-features = true
 
 [dependencies]
-aes = "0.8"
+aes = "0.7"
```

Comment:
Could you please provide more info on why is this required?

Reply:
to rollback this pr https://github.com/KomodoPlatform/librustzcash/pull/7